receiver/prometheusreceiver/config_test.go

```diff
@@ -161,3 +161,40 @@ func TestRejectUnsupportedPrometheusFeatures(t *testing.T) {
 	require.Equal(t, wantErrMsg, gotErrMsg)
 
 }
+
+func TestConfigFromFile(t *testing.T) {
+	factories, err := componenttest.NopFactories()
+	assert.NoError(t, err)
+
+	factory := NewFactory()
+	factories.Receivers[typeStr] = factory
+	cfg, err := configtest.LoadConfig(path.Join(".", "testdata", "config_from_file.yaml"), factories)
+	require.NoError(t, err)
+	require.NotNil(t, cfg)
+	err = cfg.Validate()
+	require.Nil(t, err, "Expected no error")
+}
+
+func TestConfigFromFile_nonExistentFile(t *testing.T) {
```

Comment:
```suggestion
func TestConfigFromFileNonExistentFile(t *testing.T) {
```

Reply:
Thank you @codeboten, I've addressed this change in a push.